go/install/install_linux.go

```
@@ -66,3 +67,15 @@ func KBFSBinPath(runMode libkb.RunMode, binPath string) (string, error) {
 func RunAfterStartup(g *libkb.GlobalContext, isService bool) error {
 	return nil
 }
+
+// kbfsBinName returns the name for the KBFS executable
+func kbfsBinName(runMode libkb.RunMode) (string, error) {
+	if runMode != libkb.ProductionRunMode {
+		return "", fmt.Errorf("KBFS install is currently only supported in production")
+	}
+	return "kbfsfuse", nil
+}
+
+func updaterBinName() (string, error) {
+	return "kbupdater", nil
+}
```

I think this should return an error (or panic).